docs/changelog.md

```diff
@@ -24,9 +24,20 @@ This section tracks the evolution of NEAR Treasury: find out what's new, what's
 
 **What's New ✨**
 
+- Enhanced Asset Exchange display:
```

Comment:
The new feature here would be:

- Asset Exchange for NEAR Intents leveraging the OneClick API

This includes showing info about USD values, network info etc.